css/css-syntax-3/tokenizing-and-parsing/input-byte-stream/i18n-css-syntax-bytes-001.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="ISO-8859-15"/>
+<title>CSS3 Syntax, input byte stream: CSS with UTF-signature</title>
+<link rel='author' title='Richard Ishida' href='mailto:ishida@w3.org'>
+<link rel='help' href='http://www.w3.org/TR/css3-syntax/#input-byte-stream'>
+<script src="../../../resources/testharness.js"></script>
+<script src="../../../resources/testharnessreport.js"></script>
+<link rel="stylesheet" type="text/css" href="../../resources/../testharness.css">
+<meta name='flags' content='http dom'>
+<meta name="assert" content="A UTF-8 signature at the beginning of a css style sheet sets the encoding for the style sheet to UTF-8.">
+<style type='text/css'>
+.test div { width:50px; }
+
+p { color: gray; margin-top: 3em; font-size: 80%; }
+</style>
+<link rel="stylesheet" type="text/css" href="support/bom.css">
+</head>
+<body>
+
+
+
+<div class="test"><div id="box" class="���">&#xA0;</div></div>
+
+
+<!--Notes:
+In the style sheet, the CSS selector identifies a class name with the following byte sequence: c3 bd c3 a4 c3 a8.  Read as UTF-8 this gives the characters ���.  Read as ISO 8850-15, this gives ýäè.
+
+This HTML page is served as ISO 8859-15 and uses a different byte sequence (fd e4 e8) for the class name that equates to the character sequence ���.  
+
+The class ids will therefore only match if the CSS is read as UTF-8.  (ISO 8859-15 is used for this page to avoid false matching due to UTF-8 fallback.)
+
+The CSS file encoding is established as utf-8 using only a UTF-8 signature.
+
+-->
+<script>
+test(function() {
+assert_equals(document.getElementById('box').offsetWidth, 100);
```

Comment:
Would these tests be better as reftests?

Reply:
no